modules/eks/echo-server/charts/echo-server/templates/ingress.yaml

```diff
@@ -9,20 +9,10 @@ metadata:
   name: {{ $fullName }}
   annotations:
     {{- if eq (printf "%v" .Values.ingress.nginx.enabled) "true" }}
-    kubernetes.io/ingress.class: {{ .Values.ingress.nginx.class }}
       {{- if (index .Values.ingress.nginx "tls_certificate_cluster_issuer") }}
     cert-manager.io/cluster-issuer: {{ .Values.ingress.nginx.tls_certificate_cluster_issuer }}
       {{- end }}
     {{- else if eq (printf "%v" .Values.ingress.alb.enabled) "true" }}
-    kubernetes.io/ingress.class: {{ .Values.ingress.alb.class }}
```

Comment:
why are we removing these ingress anotations?

Reply:
We do not want `echo-server` to disturb or generate any configuration any other service might rely on. So we deploy it using defaults only (as much as we can). That way there will be no ripple effect when we remove it.